Formula/conductr-cli.rb

```diff
@@ -0,0 +1,14 @@
+class ConductrCli < Formula
+  desc "The Lightbend ConductR CLI"
+  homepage "https://conductr.lightbend.com"
+  url "https://bintray.com/lightbend/generic/download_file?file_path=conductr-cli-1.0.1-Mac_OS_X-x86_64.zip"
```

Comment:
> Is the intent to signal to the user that they are indeed installing a binary?

Not exactly, since we provide binary packages ourselves (bottles) via our build infrastructure. The problem with shipping binaries *not* built by us is twofold:

1. OS X's "public" APIs/ABIs change subtly with each release, making it difficult to build a single binary that works across multiple versions.
2. As a result of 1, we (the Homebrew) maintainers would have to spend more time debugging binaries that we know nothing about (since they didn't come from us) and fielding support that upstreams should general deal with.

Inclusion in Cask better reflects the user-upstream relationship that binaries have and partially shields us from OS X-induced breakage (for a good example of this, check out our [clock_gettime tag](https://github.com/Homebrew/homebrew-core/issues?utf8=%E2%9C%93&q=label%3Aclock_gettime%20)).

Thanks for submitting and discussing, but we can't include this in core. I still recommend seeing if Cask is interested, or placing it in a custom tap - `brew` will happily install binaries from custom formulae.